.circleci/config.yml

```diff
@@ -128,6 +136,26 @@ commands:
           name: Run rubocop
           command: bundle exec rubocop
 
+  # using a more recent version of yamllink so we can use the key_duplicates rule
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.key_duplicates
+
+  lint-and-verify-helm-config:
+      description:
+      parameters:
+        environment:
+          description: Name of environment to check helm template of
+          type: string
+      steps:
+        - helm/install_helm_client:
+            version: 3.7.2
+        - *install-yamllint
```

Comment:
These steps only need doing once. Here they would be installed for each environment run, slowing things down slightly.